File: hello/exercise_serial/hello.f90
program hello
  ! Include/Use omp_lib.h/omp_lib ?
  implicit none
  integer i
  ! Add OMP Directive
  i = ! Get Thread ID
  if (mod(i,2).eq.1) then
    print *,'Hello from thread',i,', I am odd!'
  else
    print *,'Hello from thread',i,', I am even!'
  endif
  ! End OMP Directive ?
end program hello

File: hello/exercise_serial/hello.c
#include <stdio.h>
/* Include omp.h ? */
int main() {
  int id;
  /* Add Opemp pragma */
  {
    id = /* Get Thread ID */
    if (id%2==1)
      printf("Hello world from thread %d, I am odd\n", id);
    else
      printf("Hello world from thread %d, I am even\n", id);
  }
}
